letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
@pde You commented that the `${x:-y}` syntax does not work on non-POSIX busybox:
https://github.com/certbot/certbot/pull/1901#issuecomment-164976573

Does that mean that I should re-write the `${LE_AUTO_SUDO+x}` check too?